CHANGELOG.txt

```diff
@@ -109,6 +116,11 @@
 * Performance - Lazy Loading Atomic Components. #2777
 * Pefactor - Remove dashicon classes. #2848
 
+**WooCommerce Blocks 3.1.0**
+* Fix - Missing permissions_callback arg in StoreApi route definitions. #2926
+* Fix - 'Product Summary' in All Products block is not pulling in the short description of the product. #2913
+* Dev - Add query filter when searching for a table. #2886
+
```

Comment:
Can you please include changes from 4.3.1? I've just created PR also for default branch.

https://github.com/woocommerce/woocommerce/pull/27216